src/js/themes/hpe.js

```diff
@@ -2206,6 +2244,7 @@ const buildTheme = (tokens, flags) => {
       clear: {
         container: {
           pad: components.hpe.select.default.medium.drop.paddingX,
+          round: 'xsmall',
```

Comment:
I don't see this defined in base.js

Reply:
You're right -- with the latest from select clear styling we don't even need this at all actually.